packages/luna/src/components/TutorFeedback/RatingDialog.tsx

```diff
@@ -55,14 +55,14 @@ export const RatingDialog: React.FC<RateDialogProps> = ({
         studentName={studentName}
         studentId={studentId}
         imageUrl={imageUrl}
-        feedback={newFeedback}
-        rating={newRating}
+        feedback={newFeedback || feedback}
+        rating={newRating || rating || 0}
```

Comment:
This changes are not needed because we will render the dialog only if we have the rating values already.